src/components/Dashboard/TodoList.vue

```diff
@@ -893,7 +895,7 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin, E
       if (filing?.header?.status === FilingStatus.COMPLETED) {
         this.$root.$emit('reloadData')
       } else {
-        throw new Error('Filing not yet completed')
```

Comment:
The error message was ignored by the catch block below, so I removed it.

This is to avoid the [Decoy anti-pattern](https://github.com/bcgov/lear/discussions/1825#discussioncomment-4278721).